locales/locales.php

```diff
@@ -1388,6 +1388,20 @@ public function __construct() {
 		$mg->google_code = 'mg';
 		$mg->facebook_locale = 'mg_MG';
 
+		$mlt = new GP_Locale(); 
```

Comment:
Now that the key is `$mlt`, it needs to be moved after the definition of the `$ml` locale.